chef/cookbooks/provisioner/templates/default/crowbar_join.suse.sh.erb

```diff
@@ -163,9 +167,9 @@ do_setup() {
     # Make sure that the client knows how to talk to the server.
     local cfg=/etc/chef/client.rb
     if ! [ -f $cfg ] || \
-       ! grep -q "^\s*chef_server_url\s*[\"\']http://$IP:4000[\"\']" $cfg; then
+       ! grep -q "^\s*chef_server_url\s*[\"\']http://$IP_WRAPPED:4000[\"\']" $cfg; then
```

Comment:
And this? Because below is ADMIN_IP_WRAPPED, not IP_WRAPPED ...

Reply:
huh, it's all IP_WRAPPED in this script. Unless I'm missing something.. which might be possible :P